draft-ietf-tls-tls13.md

```diff
@@ -3650,15 +3667,6 @@ close_notify
   any more messages on this connection. Any data received after a
   closure MUST be ignored.
 
-end_of_early_data
```

Comment:
[This bit was even kind of a lie before. end_of_early_data did not signal "that the connection is ending".]